thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/function/UserRuleAnomalyFunction.java

```diff
@@ -40,14 +41,48 @@
   public static final String BASELINE = "baseline";
   public static final String CHANGE_THRESHOLD = "changeThreshold";
   public static final String AVERAGE_VOLUME_THRESHOLD = "averageVolumeThreshold";
+  public static final String MIN_CONSECUTIVE_SIZE = "minConsecutiveSize";
   public static final String DEFAULT_MESSAGE_TEMPLATE = "threshold=%s, %s value is %s / %s (%s)";
   private static final Joiner CSV = Joiner.on(",");
+  private static final Joiner ANOMALY_RESULT_MESSAGE_JOINER = Joiner.on("...");
```

Comment:
Let's discuss how to format the message for anomalies that end up merged. Simply joining them together can result in a lengthy and complicated message that might be more confusing than helpful for users.